src/main/java/org/myrobotlab/service/WebGui.java

```diff
@@ -989,28 +989,26 @@ public void startService() {
     try {
       extract();
     } catch (Exception e) {
-      Logging.logError(e);
+      log.error("webgui start service threw", e);
     }
 
     start();
   }
 
   public void stop() {
     if (nettosphere != null) {
-
-      log.info("stopping nettosphere");
+      log.warn("==== nettosphere STOPPING ====");
+      nettosphere.stop();
       // Must not be called from a I/O-Thread to prevent deadlocks!
-      (new Thread("stopping nettophere") {
-        public void run() {
-          /*
-           * nettosphere.framework().removeAllAtmosphereHandler();
-           * nettosphere.framework().resetStates();
-           * nettosphere.framework().destroy();
-           */
-          nettosphere.stop();
-        }
-      }).start();
-      sleep(1000);
+      /*
+       * (new Thread("stopping nettophere") { public void run() {
+       * 
+       * log.error("==== nettosphere stopping  ===="); //
+       * nettosphere.framework(). // nettosphere.framework().destroy();
+       * nettosphere.stop(); log.error("==== nettosphere STOPPED ===="); }
+       * }).start(); sleep(1000);
+       */
+      log.warn("==== nettosphere STOPPED ====");

```

Comment:
this should also be info level.    (only if the stop() method blows up. should we print a warn/stack trace..)  but it doesn't look like stop() can throw anything.